src/bidiMapper/domains/script/scriptEvaluator.ts

```diff
@@ -94,7 +94,7 @@ export class ScriptEvaluator {
     resultOwnership: Script.ResultOwnership,
     serializationOptions: Script.SerializationOptions
   ): Promise<Script.ScriptResult> {
-    if (![0, undefined].includes(serializationOptions?.maxDomDepth))
+    if (![0, undefined, null].includes(serializationOptions.maxDomDepth))
```

Comment:
```suggestion
    if (![0, null, undefined].includes(serializationOptions.maxDomDepth))
```

Let's use the same order as below for consistency (no particular preference about the order, you could also change it the other way if preferred)